src/app/public/modules/dropdown/dropdown.component.spec.ts

```diff
@@ -46,20 +46,20 @@ describe('Dropdown component', () => {
     fixture.destroy();
   });
 
-  function openPopoverWithButtonClick() {
+  function openOrClosePopoverWithButtonClick(isClose: boolean = false) {
```

Comment:
Since `if (isClose)` evaluates the same when `isClose` is `undefined` or `false`, I would remove the default value and simply declare it as optional (`isClose?: boolean`).

Reply:
it isn't used in a conditional though, it is passed to `verifyMenuVisibility(isVisible: boolean`) which has a couple of expect statements that check if values are equal to `isVisible`. I could change those check to be `toBeTruthy()`/`toBeFalsy()` based on the value of `isVisible`, but I think it is cleaner to just leave the default value for `openOrClosePopoverWithButtonClick`.

Reply:
Makes sense.